aten/src/ATen/native/cuda/ReduceOpsKernel.cu

```diff
@@ -23,6 +23,13 @@ struct SimpleCopy {
   }
 };
 
+template <typename scalar_t>
+struct AbsCopy {
```

Comment:
Where is this used?

Reply:
This is a lost child from previous rebase. This function is obsolete now, will remove it.